docs/topics/settings.rst

```diff
@@ -1473,7 +1473,67 @@ If a reactor is already installed,
 
 :meth:`CrawlerRunner.__init__ <scrapy.crawler.CrawlerRunner.__init__>` raises
 :exc:`Exception` if the installed reactor does not match the
-:setting:`TWISTED_REACTOR` setting.
+:setting:`TWISTED_REACTOR` setting. Top-level
+:mod:`~twisted.internet.reactor` imports in project files & imported
+3rd party libraries may also raise :exc:`Exception` as Twisted will install the
+default reactor before scrapy installs reactor specified in
+:setting:`TWISTED_REACTOR` leading to a mismatch in installed reactor.
```

Comment:
Those top-level imports won't raise an exception, Scrapy will, when it checks which reactor is installed, so this can be rephrased.

Reply:
Okay I will change it to:
`CrawlerRunner` raises `Exception` if the installed reactor does not match the `TWISTED_REACTOR` setting; therefore, having top-level `twisted.internet.reactor` imports in project files & imported 3rd party libraries will make Scrapy raise `Exception` when it checks which reactor is installed.